client/api/blocks.js

```diff
@@ -0,0 +1,77 @@
+import $ from 'jquery';
```

Comment:
This file was added to separate the UPE/combined API calls entirely from the Blocks API calls.